data/css/_colors.scss

```diff
@@ -1,3 +1,5 @@
+$brand-color: #005a71 !default;
```

Comment:
I need 10 different colors to style this template. We currently only offer 7, so I have to add more. I have no preference as to what the new color variables are called - I tried to follow the names design was giving them in the specs.